### PR TITLE
[fix] Update empty check since text node can be an atom

### PR DESCRIPTION
**Pull request by velveret**
Placeholder wasn't showing for " " content (the default) since the text node was passing the `isAtom` case.

Fix:
image

### CODE REVIEW

File: dist/ContentPlaceholderPlugin.js [hunk 1]
```diff
@@ -156,19 +156,16 @@ var ContentPlaceholderView = function () {
     key: '_show',
     value: function _show() {
       var el = this._el;
-      if (el && this._visible !== true) {
-        this._visible = true;
```
Comment (hedgerwang) [inline dist/ContentPlaceholderPlugin.js]: Is there any good reason that you want to drop `this._visible` and use `el.style.display` instead?

Usually it's relatively inefficient and unsafe to use DOM element as source of truth for state management. 

Ideally DOM element should only be used as a pure view that reflects the managed state.

Thought this change does not change the behavior, it does change the idea behind the implementation. 

Unless you have strong reason that this should change, I'd recommend reverting this file.

Comment (velveret) [reply dist/ContentPlaceholderPlugin.js]: I was running into a race condition where the editor state would be from the previous comment, and `_visible` would be set to `false` when the placeholder was still shown.

Screen recording of the bug (placeholder still shown while editing second comment):
comment-placeholder-bug

**Comment by velveret** [comment]
Discussed offline - we're going to hold off on landing, since the race condition may affect other plugins as well.

**Comment by velveret** [comment]
As discussed offline, reverted the second fix related to placeholder visibility, will push as a separate branch

**Pull request merged**